landscape/client/broker/client.py

```diff
@@ -142,11 +142,13 @@ def _run_with_error_log(self):
 
     def _error_log(self, failure):
         """Errback to log and reraise uncaught run errors."""
-        msg = "{} raised an uncaught exception".format(type(self).__name__)
+        cls = type(self).__name__
+        msg = f"{cls} raised an uncaught exception"
         if sys.exc_info() == (None, None, None):
             error(msg)
         else:
             exception(msg)
+        debug(str(failure.value))
```

Comment:
Maybe the traceback module would be useful here?

Reply:
Yup. Good idea. I'll use it.

Reply:
Actually it seems that `traceback` may not play nicely with the `errBack` used in Twisted. Checking it out.